# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/gnome2/galeon/compile.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


The usual trival compile fixes for lazy people. (Yes, this is really
a 1:1 copy of parts of my devhelp patch ...)

  - Rene Rebe <rene@rocklinux.org>

--- devhelp/libegg/eggtoolbar.c.orig	2004-05-14 10:32:25.233789992 +0200
+++ devhelp/libegg/eggtoolbar.c	2004-05-14 10:32:51.857742536 +0200
@@ -2570,7 +2570,7 @@
 
 GtkWidget*
 egg_toolbar_append_element (EggToolbar          *toolbar,
-			    GtkToolbarChildType  type,
+			    EggToolbarChildType  type,
 			    GtkWidget           *widget,
 			    const char          *text,
 			    const char          *tooltip_text,
@@ -2587,7 +2587,7 @@
 
 GtkWidget *
 egg_toolbar_prepend_element (EggToolbar          *toolbar,
-			     GtkToolbarChildType  type,
+			     EggToolbarChildType  type,
 			     GtkWidget           *widget,
 			     const char          *text,
 			     const char          *tooltip_text,
